portfolio/src/main/webapp/script.js

```diff
@@ -34,9 +34,19 @@ async function getContent() {
 }
 
 function getComments(){
-   String json = fetch('/data')
    response.setContentType("text/html;");//going to priint directly on page
    response.getWriter().println(json);;
-  }
+   fetch('/data').then(response => response.json()).then((list) => {
 
+    const commentList = document.getElementById('');
+    statsListElement.innerHTML = '';
```

Comment:
where is the `statsListElement` defined?

Reply:
Deleted this portion and used the framework to create the current function.